src/main/webapp/resources/css/inward_bill_service.css

```
@@ -0,0 +1,98 @@
+    display: flex;
+    flex-direction: column;
+}
+@media print {
+
+    .fiveinchbill{
+        width: 14.5cm !important;
+        min-height: 14.2cm !important;
+        margin: 0 auto !important;
+        padding: 6px !important;
+
+        border: none !important;      /* Remove outer border */
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)